src/components/transactions/TxDetails/Summary/TxDataRow/index.tsx

```diff
@@ -15,7 +15,9 @@ export const TxDataRow = ({ title, children }: TxDataRowProps): ReactElement | n
   if (children == undefined) return null
   return (
     <div className={css.gridRow}>
-      <div className={css.title}>{title}</div>
```

Comment:
You could keep the div and just use `color: var(--color-text-secondary);` in the CSS file.

Reply:
on it.